packages/dd-trace/src/plugins/util/inferred_proxy.js

```diff
@@ -57,13 +57,18 @@ function createInferredProxySpan (headers, childOf, tracer, context) {
   )
 
   tracer.scope().activate(span)
-  context.inferredProxySpan = span
+  reqCtx.inferredProxySpan = span
   childOf = span
 
   log.debug('Successfully created inferred proxy span.')
 
   setInferredProxySpanTags(span, proxyContext)
 
+  if (traceCtx) {
```

Comment:
This is repeated in a few places at this point. Could we maybe create a `WebPlugin` that would reuse the existing code instead in addition to call the util?